modules/loader/src/main/resources/application.conf

```diff
@@ -8,6 +8,10 @@
         "strategy": "EXPONENTIAL",
         "attempts": 3
     },
+    "readyCheck": {
+        "backoff": "30 seconds",
+        "strategy": "EXPONENTIAL"
```

Comment:
Also, exponential seems to be not very good strategy. I think something like reverse exponential is even better, because we know it _will_ become available and every new check is getting us closer, so we want to shring the pauses, not grow them.

Exponential is good when we don't know if server will ever respond and we just bombarding it with requests less often.

Reply:
Seems I lost one comment. Exponential can actually be a good default for Redshift. Because if Redshift is unavailable for use - it likely means there's something wrong (and long) happening with it, such as resizing.

Reply:
Seems like `constant` and some small backoff duration (15 seconds ?) would be suitable for all destinations as default. Since there will be only one loader instance, I guess it shouldn't be too much problem sending `select 1` every 15 seconds. WDYT?